roseau/load_flow/models/loads/loads.py

```diff
@@ -22,7 +22,7 @@ class AbstractLoad(Element, metaclass=ABCMeta):
     _power_load_class: type["PowerLoad"]
     _current_load_class: type["CurrentLoad"]
     _impedance_load_class: type["ImpedanceLoad"]
-    _flexible_load_class: type["FlexibleLoad"]
+    _flexible_parameter_class = FlexibleParameter
```

Comment:
We will need to check in January with @Saelyos to see if you can define it here directly. That's sounds good for now.